### ISSUE TITLE
SAC trained model gives error in 0.10: Requires an unknown input named: other_recurrent_in

### ISSUE THREAD

**Issue by DooblyNoobly** [Unity-Technologies/ml-agents#2695]
**Describe the bug**
When using a SAC trained model in ML Agents 0.10 in inference mode, a warning is shown in the editor and an error thrown at runtime.

**To Reproduce**
1. Train a model using SAC trainer
2. Drag the model into the brain in the editor and notice the warning
3. Press play in the editor
4. Notice the errors:
![image](https://user-images.githubusercontent.com/20813925/66367431-76b00b80-e9d7-11e9-9dad-2628a4b7428a.png)

Attached is the config I was using for the training:
```
default:
    trainer: sac
    batch_size: 32
    beta: 0.00455000018700957
    buffer_size: 1000000
    buffer_init_steps: 1000
    init_entcoef: 0.1
    train_interval: 3
    vis_encode_type: simple
    epsilon: 0.216
    hidden_units: 64
    lambd: 0.9144
    learning_rate: 0.000155999994603917
    tau: 0.005
    max_steps: Infinity
    memory_size: 64
    normalize: True
    num_epoch: 3
    num_layers: 3
    num_update: 1
    time_horizon: 32
    sequence_length: 16
    summary_freq: 4000
    use_recurrent: True
    reward_signals:
        extrinsic:
            strength: 1
            gamma: 0.99
    curiosity:
        strength: 0.02
        gamma: 0.99
        encoding_size: 256
```

**Comment by ervteng** [comment]
Hey @DooblyNoobly, we've logged this issue with internal tracking number MLA-106. We're working on a fix, expect a patch soon.

**Comment by ervteng** [comment]
@DooblyNoobly, this issue was fixed in the latest Hotfix 0.10.1 - thanks for reporting and closing the issue for now.
